go/host/interfaces.go

```diff
@@ -26,7 +26,7 @@ type Host interface {
 	// Subscribe feeds logs matching the encrypted log subscription to the matchedLogs channel.
 	Subscribe(id rpc.ID, encryptedLogSubscription common.EncryptedParamsLogSubscription, matchedLogs chan []byte) error
 	// Unsubscribe terminates a log subscription between the host and the enclave.
-	Unsubscribe(id rpc.ID) error
```

Comment:
I felt that logging the error in the method was better than returning an error. There's not much the consumer can do if the unsubscribe fails.